tests/test_adapter_stac.py

```diff
@@ -51,6 +52,32 @@ def test_invocation_processes_items_with_sources(self):
         self.assertEqual(AdapterTester.process_args[0][1], message.sources[0])
         self.assertEqual(AdapterTester.process_args[1][1], message.sources[0])
 
+    @patch('harmony.adapter.read_file')
+    def test_invocation_follows_linked_catalogs(self, test_patch):
+        catalog0 = Catalog('0', 'Catalog 0')
+        catalog0.add_link(Link('harmony_source', 'http://example.com/C0001-EXAMPLE'))
+        catalog1 = Catalog('1', 'Catalog 1')
+        catalog1.add_link(Link('harmony_source', 'http://example.com/C0002-EXAMPLE'))
+        catalog0.add_link(Link('next', 'catalog1.json'))
```

Comment:
Maybe add a prev link in one of these tests to another catalog to make sure it isn't followed and doesn't cause any problems?

Reply:
Done.